ts/defs.bzl

```diff
@@ -433,7 +433,7 @@ def ts_project(
         srcs = srcs,
         args = args + select(
             {
-                "//ts:skip_lib_check.always": ["--skipLibCheck"],
+                "@rules_ts//ts:skip_lib_check.always": ["--skipLibCheck"],
                 "//ts:skip_lib_check.honor_tsconfig": [],
```

Comment:
this line needs it too.

Reply:
yeah that explains the CI failure I was ignoring :)